modules/AlphaWalletFoundation/AlphaWalletFoundation/Core/CoinTicker/CoinGecko/CoinGeckoNetworkProvider.swift

```diff
@@ -86,9 +86,13 @@ public class CoinGeckoNetworkProvider: CoinGeckoNetworkProviderType {
         return provider.publisher(.pricesOfTokens(ids: tickerIds, currency: Constants.Currency.usd, page: page), callbackQueue: .global())
             .retry(times: 3)
             .tryMap { [decoder] response -> [CoinTicker] in
-                return try response.map([CoinTicker].self, using: decoder)
+                do {
```

Comment:
We can remove the `do-catch` right? `try` already throws. (maybe this is after removing the debug log? :) )